tool/write-ci-info.sh

```diff
@@ -4,53 +4,15 @@ if [[ $TASK != *build* ]]; then exit; fi
 
 FILE=src/_data/ci.yaml
 
-echo "# WARNING: the sample values in this file get regenerated at build time.
-build-id: $TRAVIS_BUILD_ID
-build-number: $TRAVIS_BUILD_NUMBER
+echo "# WARNING: the values in this file get regenerated at build time.
+build-id: ${GITHUB_RUN_ID:-$TRAVIS_BUILD_ID}
+build-number: ${GITHUB_RUN_NUMBER:-$TRAVIS_BUILD_NUMBER}
 build-time: $(date)
-commit-sha: $TRAVIS_COMMIT
-job-id: $TRAVIS_JOB_ID
-repo-slug: $TRAVIS_REPO_SLUG" > $FILE
+commit-sha: ${$GITHUB_SHA:-$TRAVIS_COMMIT}
+job-id: ${GITHUB_ACTION:-$TRAVIS_JOB_ID}
+repo-slug: ${GITHUB_REPOSITORY:-$TRAVIS_REPO_SLUG}" > $FILE
 
 if [[ "$1" == '-v' ]]; then
   echo "$FILE:"
   cat $FILE
 fi
-
```

Comment:
This list of sample values is prone to be outdated, so I removed it. The file is probably rather stable and the default environment is document in the help of whatever CI system is used.

Reply:
sgtm